include/stop_token.hpp

```diff
@@ -126,14 +126,14 @@ namespace std {
     template <class>
       friend class in_place_stop_callback;
 
-    uint8_t __lock_() noexcept;
-    void __unlock_(uint8_t) noexcept;
+    inline uint8_t __lock_() noexcept;
+    inline void __unlock_(uint8_t) noexcept;
```

Comment:
`inline` should not be necessary on the forward declarations, only on the definitions.

```suggestion
    uint8_t __lock_() noexcept;
    void __unlock_(uint8_t) noexcept;
```